### PR TITLE
[HER-100] Create pagination graph type

### PR DESCRIPTION
**Pull request by ConstantinescuAndrei**
## Description
- it will contain page size and page index fields

## Related
[HER-100](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?issueParent=10022&selectedIssue=HER-100)

### CODE REVIEW

File: src/Presentation/Pagination/Types/TPPagination.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    using Business.Pagination;
+    using global::GraphQL.Types;
+    using Style;
+
+    [SuppressMessage(Category.Default, Check.CA1724, Justification = Reason.Readability)]
+    public sealed class TPPagination : ObjectGraphType<PPagination>
+    {
+        public TPPagination()
+        {
+            Field(pagination => pagination.PageSize).Description("This is the page size");
```
Comment (danidoff) [inline src/Presentation/Pagination/Types/TPPagination.cs:13]: Should we also add the Name and Description for this type ?

File: src/Presentation/Pagination/Types/TPPagination.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+namespace Presentation.Pagination.Types
+{
+    using System.Diagnostics.CodeAnalysis;
+    using Business.Pagination;
+    using global::GraphQL.Types;
+    using Style;
+
+    [SuppressMessage(Category.Default, Check.CA1724, Justification = Reason.Readability)]
+    public sealed class TPPagination : ObjectGraphType<PPagination>
```
Comment (danidoff) [inline src/Presentation/Pagination/Types/TPPagination.cs:9]: Should be an InputObjectGraphType?

**Review by danidoff — CHANGES_REQUESTED** [review]

File: src/Presentation/Pagination/Types/TPPagination.cs (hunk 2 above)
Comment (ConstantinescuAndrei) [reply src/Presentation/Pagination/Types/TPPagination.cs]: done

File: src/Presentation/Pagination/Types/TPPagination.cs (hunk 1 above)
Comment (ConstantinescuAndrei) [reply src/Presentation/Pagination/Types/TPPagination.cs]: done

**Review by andreea-dumitrascu — DISMISSED** [review]

File: src/Presentation/Pagination/Types/TPPagination.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    using global::GraphQL.Types;
+    using Style;
+
+    [SuppressMessage(Category.Default, Check.CA1724, Justification = Reason.Readability)]
+    public class TPPagination : InputObjectGraphType<PPagination>
+    {
+        public TPPagination()
+        {
+            Name = "Pagination";
+            Description = "It will be used for fetching data for a single page";
```
Comment (danidoff) [inline src/Presentation/Pagination/Types/TPPagination.cs:14]: Should we use a better naming for Name and Description like `Name = "PaginationParameter"` and `Description = "The parameter used to specify the pagination info"`?

**Review by danidoff — CHANGES_REQUESTED** [review]

**Review by Urzica-Platon-Casian — APPROVED** [review]
Zexy!

**Review by andreea-dumitrascu — APPROVED** [review]

**Review by RaduBuciuceanu — APPROVED** [review]

**Review by danidoff — APPROVED** [review]

**Pull request merged**